Samples/CayenneDecoderTest/CayenneTest.cs

```diff
@@ -0,0 +1,189 @@
+using System;
+using Xunit;
+using CayenneDecoderModule.Classes;
+
+namespace CayenneDecoderTest
+{
+    public class CayenneTest
+    {
+        [Fact]
+        public void Temeperature()
+        {
+            CayenneEncoder cayenneEncoder = new CayenneEncoder();
+            double temp = 33.7;
+            byte channel = 1;
+            cayenneEncoder.AddTemperature(channel, temp);
+            var buff = cayenneEncoder.GetBuffer();
+            CayenneDecoder cayenneDecoder = new CayenneDecoder(buff);            
+            cayenneDecoder.TryGetTemperatureSensor(out TemperatureSensor temperatureSensor);
+            Assert.True(temperatureSensor.Value == temp);
```

Comment:
For comparison use Assert.Equal(temp, temperatureSensor.Value);

Reply:
ok, will correct. thanks